### ISSUE TITLE
[FRS] Video Production Workshop Series

### ISSUE THREAD

**Issue by sohdesune** [OpenSUTD/Discovery-Week-Working-Title-#5]
Productions 101

Want to upgrade from Windows Movie Maker? Take our crash course in film production, covering ideating to filming to editing!

by SUTD Productions

Participants will learn about scriptwriting, camera work, and video and audio editing using _Adobe Premiere Pro and related software in the Adobe Creative Cloud_. Participants do not need prior experience as this is a basic workshop series that will end with a small short film competition to spark their curiosity in film and boost their interest in joining SUTD Productions. 

We can handle up to around 30 participants and intend to conduct the workshops at the iDiA Lab (2.605). We can book the venue by ourselves. We will need to conduct our session in the evening due to the availability of seniors. If possible, we hope to have our slot on Monday, 27 May (1800-2100 hrs).

The workshop series will be facilitated by seniors in the club, some of which have experience in external semi-professional projects.

We will not require additional resources beyond our internal club assets.

Do contact us at productions@club.sutd.edu.sg

Thank you!

**Comment by Fishbiscuit** [comment]
**Video Production Workshop Series**

**Abstract**
Want to upgrade from Windows Movie Maker? Take our crash course in film production, covering ideating to filming to editing!

Participants will learn about scriptwriting, camera work, and video and audio editing using Adobe Premiere Pro and related software in the Adobe Creative Cloud. Participants do not need prior experience as this is a basic workshop series that will end with a small short film competition to spark their curiosity in film and boost their interest in joining SUTD Productions.

**Student Organisation organising this session**
by SUTD Productions

**Session intent (what to expect)**
- Scriptwriting
- Video and audio editing using *Adobe Premiere Pro and related software in Adobe Creative Cloud*
- workshop series ending with a small short film competition

**Session details**
We can handle up to around 30 participants and intend to conduct the workshops at the iDiA Lab (2.605). We can book the venue by ourselves. We will need to conduct our session in the evening due to the availability of seniors. If possible, we hope to have our slot on Monday, 27 May (1800-2100 hrs).

**Speakers/Workshop facilitators**

The workshop series will be facilitated by seniors in the club, some of which have experience in external semi-professional projects.

**Logistical details**
We will not require additional resources beyond our internal club assets.



E-mail: productions@club.sutd.edu.sg

**Comment by Fishbiscuit** [comment]
Could you check if this is accurate? Also it would help greatly if you could let us know how many sessions do you expect to hold please

Otherwise thank you and it looks like an exciting session coming up!

**Comment by sohdesune** [comment]
Hi, sorry for the slow response. Yes it is correct. We expect to run weekly sessions until recess week (i.e. 5-6 sessions). If possible, perhaps we could fill both Monday slots for Discovery Week?

Thank you!

**Comment by sohdesune** [comment]
Hi @Fishbiscuit can I check if the session is confirmed?

Thanks!

**Comment by Fishbiscuit** [comment]
yes it is! we're in the midst of collating all overall sessions and setting up the signup process
